caom2-search-lib/src/main/java/ca/nrc/cadc/search/upload/StreamingIterator.java

```diff
@@ -134,6 +141,7 @@ public Element next() throws NoSuchElementException
         tableRow.addContent(createTableData(ra));
         tableRow.addContent(createTableData(dec));
         tableRow.addContent(createTableData(radius));
+        tableRow.addContent(createTableData(position));
```

Comment:
Can the RA, Dec, and Radius be removed from the upload table now?  Or are we preserving them for the original values?

Reply:
RA and Dec are still displayed in the results table, for both single target and target upload files, they are in the default columns displayed. 

The position strings when they return have 16 digit precision that adds a '2' on... I'm not sure if that's going to be appropriate for cutout use, if not then the radius' will be needed.

Reply:
You're referring to the `Upload Target RA` and `Upload Target Dec`?  If that's the case and uses those columns then please leave them in.

Reply:
ok. didn't make any changes here.